src/System.CommandLine.Subsystems/ConsoleHack.cs

```diff
@@ -10,7 +10,7 @@ public class ConsoleHack
     private readonly StringBuilder buffer = new();
     private bool redirecting = false;
 
-    public void WriteLine(string text)
+    public void WriteLine(string text = "")
```

Comment:
Is empty string actually a valid default?

Reply:
I think I would re-write WriteLine to act slightly different here if this was going to stay and to behave more like Console.WriteLine, but I think ConsoleHack is going to disappear so we can ignore this

Reply:
It's valid, at least in current `main` behavior. See my other comment for the code that uses this call, but this was done to preserve the existing `WriteLine()` interface for writing an empty newline